src/qt/bitcoinunits.cpp

```diff
@@ -38,11 +39,12 @@ bool BitcoinUnits::valid(int unit)
 
 QString BitcoinUnits::name(int unit)
 {
+    const QString baseUnit = QString::fromStdString(CURRENCY_UNIT);
     switch(unit)
     {
-    case BTC: return QString("BTC");
-    case mBTC: return QString("mBTC");
-    case uBTC: return QString::fromUtf8("μBTC");
+    case BTC: return QString("%1").arg(baseUnit);
```

Comment:
I see this was the goal, but you must have `::fromUtf8` in the micro case anyway...